auacm/app/modules/submission_manager/judge.py

```diff
@@ -195,16 +196,16 @@ def _execute_submission(self):
 
                 # The execution is completed.  Check its correctness.
                 with open(os.path.join(
-                          self.prob_output_path, out_file)) as correct, \
+                    self.prob_output_path, out_file)) as correct, \
                      open(os.path.join(
-                          self.sub_output_path, out_file)) as sub_result:
+                         self.sub_output_path, out_file)) as sub_result:
                     correct_lines = correct.readlines()
                     submission_lines = sub_result.readlines()
                     if len(submission_lines) != len(correct_lines):
                         return WRONG_ANSWER, test_number, max_time
 
-                    for gl, sl in zip(correct_lines, submission_lines):
-                        if gl.rstrip('\r\n') != sl.rstrip('\r\n'):
+                    for key, value in zip(correct_lines, submission_lines):
```

Comment:
I feel like `key` and `value` are not very descriptive. Maybe `judge_line` / `expected_line` and `submission_line` / `actual_line` or something to that effect.